core-plugins/src/test/java/co/cask/hydrator/plugin/batch/aggregator/RowDenormalizerAggregatorTest.java

```diff
@@ -158,12 +161,22 @@ public void testDenormalizerWithMultipleKeyFieldValues() throws Exception {
         Assert.assertEquals("PQR1 place near XYZ1", record.get("Addr").toString());
       }
     }
+
+    DataSetManager<Table> recordsManager = getDataset(errorDatasetName);
```

Comment:
Looks like this functionality can be moved into a method? 
``private int getErrorDatasetSize(String errorDatasetName)``